README.md

```diff
@@ -372,6 +373,20 @@ Available `options`:
 * `ttr` - time to release in seconds. The time after which, if there is no active
 connection in the session, it will be released with all its tasks.
 
+## Session identify
```

Comment:
I'm not sure, but maybe it worth to name the section 'Overcome a reconnect'. That is what a user will actually find.